### ISSUE TITLE
Test fails on windows

### ISSUE THREAD

**Issue by cdietrich** [eclipse/xtext-xtend#1114]
BuildAffectionTest fails on windows i assume we need to harmonize line endings in expectation
when run from maven there are a lot like these

```
Tests run: 2, Failures: 0, Errors: 2, Skipped: 0, Time elapsed: 1.288 s <<< FAILURE! - in org.eclipse.xtend.ide.tests.buildpath.XtendLibClasspathAdderTest
org.eclipse.xtend.ide.tests.buildpath.XtendLibClasspathAdderTest  Time elapsed: 1.288 s  <<< ERROR!
org.eclipse.core.internal.resources.ResourceException: Problems encountered while deleting resources.
```
e.g.
```
org.eclipse.xtend.ide.tests.quickfix.EqualsWithNullMultiQuickfixTest.equals_with_null_in_expression(org.eclipse.xtend.ide.tests.quickfix.EqualsWithNullMultiQuickfixTest)
  Run 1: EqualsWithNullMultiQuickfixTest.setUp:150->AbstractWorkbenchTest.setUp:47 ╗ Resource
  Run 2: EqualsWithNullMultiQuickfixTest>AbstractWorkbenchTest.tearDown:54 ╗ Resource P...

org.eclipse.xtend.ide.tests.quickfix.EqualsWithNullMultiQuickfixTest.equals_with_null_in_switch(org.eclipse.xtend.ide.tests.quickfix.EqualsWithNullMultiQuickfixTest)
  Run 1: EqualsWithNullMultiQuickfixTest.setUp:150->AbstractWorkbenchTest.setUp:47 ╗ Resource
  Run 2: EqualsWithNullMultiQuickfixTest>AbstractWorkbenchTest.tearDown:54 ╗ Resource P...

org.eclipse.xtend.ide.tests.quickfix.QuickfixTest.org.eclipse.xtend.ide.tests.quickfix.QuickfixTest
  Run 1: QuickfixTest>AbstractXtendUITestCase.setUpProject:124 ╗ Resource Problems enco...
  Run 2: QuickfixTest>AbstractXtendUITestCase.tearDownProject:130 ╗ Resource Problems e...

```

**Comment by cdietrich** [comment]
org.eclipse.xtend.ide.tests.buildpath.AbstractJunitLibClasspathAdderTestCase.assertRequireBundles(String[])
has a resource leak that might cause the problem

```
Status ERROR: org.eclipse.core.filesystem code=273 Problems encountered while deleting files. null children=[Status ERROR: org.eclipse.core.filesystem code=273 Could not delete: C:\xtext-master\git\xtext-xtend\org.eclipse.xtend.ide.tests\target\work\data\test.project\META-INF\MANIFEST.MF. java.nio.file.FileSystemException: C:\xtext-master\git\xtext-xtend\org.eclipse.xtend.ide.tests\target\work\data\test.project\META-INF\MANIFEST.MF: Der Prozess kann nicht auf die Datei zugreifen, da sie von einem anderen Prozess verwendet wird.
 Status ERROR: org.eclipse.core.filesystem code=273 Could not delete: C:\xtext-master\git\xtext-xtend\org.eclipse.xtend.ide.tests\target\work\data\test.project\META-INF. null]
```

**Comment by cdietrich** [comment]
fixed in 2.24
